.github/workflows/playground.yml

```
@@ -34,10 +35,24 @@ jobs:
           pnpm install --frozen-lockfile
           pnpm build
           pnpm playground
-      - name: Deploy 🚀
+      - name: Deploy Main 🚀
+        if: github.ref == 'refs/heads/main'
         uses: JamesIves/github-pages-deploy-action@v4.7.2
         with:
           folder: packages/playground/out
           branch: previews
-          clean-exclude: pr/ # Ensure that we don't delete the pr previews
+          clean-exclude: | # Ensure that we don't delete the pr & dev previews
+            dev/
+            pr/
           force: false # Ensure that we don't override the pr previews
+      - name: Deploy Dev 🚀
+        if: github.ref == 'refs/heads/development'
+        uses: JamesIves/github-pages-deploy-action@v4.7.2
+        with:
+          folder: packages/playground/out
+          branch: previews
+          clean-exclude: | # Ensure that we don't delete the pr & dev previews
+            dev/
+            pr/
```

AFAIK this will result in the `main` deployment being deleted, as the `clean` deletes all files that aren't listed in `clean-exlude`. We should push the changes from `main` to its own `main` folder:
```suggestion
          clean-exclude: | # Ensure that we don't delete the pr & main previews
            main/
            pr/
```

---

Yep good point, I'll put that in.